app/src/main/java/org/android/go/sopt/ui/mypage/MyPageViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package org.android.go.sopt.ui.mypage
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import org.android.go.sopt.util.User
+
+class MyPageViewModel : ViewModel() {
+    private val _isLogin = MutableLiveData<Boolean>(User.isLoggedIn.value)
+    val isLogin: LiveData<Boolean> = _isLogin
```

Comment:
```suggestion
    val isLogin: LiveData<Boolean>
        get() = _isLogin
```
getter를 설정해주어야 변화하는 _isLogin 값을 사용할 수 있을 것 같습니다!